core/src/main/java/com/klaytn/caver/methods/response/KlayBlock.java

```diff
@@ -388,4 +399,24 @@ public Block deserialize(
             }
         }
     }
+
+    static class TransactionsDeserializer extends JsonDeserializer<List> {
+
+        @Override
+        public List deserialize(
+                JsonParser jsonParser, DeserializationContext deserializationContext)
+                throws IOException {
+
+            ObjectMapper objectMapper = (ObjectMapper) jsonParser.getCodec();
+            ArrayNode root = objectMapper.readTree(jsonParser);
+
+            if (root.get(0).isTextual()) {
+                return (List) objectMapper.convertValue(root, new TypeReference<List<String>>() {
+                });
+            } else {
```

Comment:
Did you test the situation that 'transactions' is empty? If you did, is it worked as well?

Reply:
Good point. If empty, `root.get(0)` cannot be called because root size is 0.